checklists/apim_checklist.en.json

```diff
@@ -204,13 +264,23 @@
         {
             "category": "Management",
             "subcategory": "Performance and scalability",
-            "service": "APIM",
             "waf": "Performance",
+            "service": "APIM",
             "text": "Deploy self-hosted gateways where Azure doesn't have a region close to the backend APIs.",
             "guid": "84b94abb-59b6-4b9d-8587-3413669468e8",
             "severity": "Medium",
             "link": "https://learn.microsoft.com/azure/api-management/api-management-howto-provision-self-hosted-gateway"
         },
+        {
+            "category": "Management",
+            "subcategory": "Failover and Caching",
+            "text": "Use Policies to add a fail-over backend URL and caching to reduce failing calls.",
+            "waf": "Reliability",
+            "service": "APIM",
+            "guid": "43e60b94-7bca-43a2-aadf-efb04d63a485",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/azure/api-management/api-management-howto-deploy-multi-region#-route-api-calls-to-regional-backend-services"
```

Comment:
I think a more applicable link for this would be the documentation for the Retry policy, where we have some examples how to perform fail-over to backend URLs:

https://learn.microsoft.com/en-us/azure/api-management/retry-policy

Reply:
Let me add the link, we can always remove it if we all agree. I guess the person that added to the mind map had something in mind...